packages/domain-picker/src/domain-picker/locked-purchased-item.tsx

```diff
@@ -0,0 +1,19 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+import { __ } from '@wordpress/i18n';
+import { Icon, check } from '@wordpress/icons';
+
+const LockedPurchasedItem: FunctionComponent< { domainName: string } > = ( { domainName } ) => {
+	return (
+		<div className="domain-picker__suggestion-item locked">
+			<div className="domain-picker__suggestion-item-name">{ domainName }</div>
+			<div className="domain-picker__price">
+				<Icon icon={ check } size={ 18 } /> { __( 'Purchased', __i18n_text_domain__ ) }
+			</div>
+		</div>
+	);
+};
+
+export default LockedPurchasedItem;
```

Comment:
This component doesn't seem to have anything to do with `domain-picker` package. We need the exact same UI component also for `plans-grid`.
<img width="520" alt="Screenshot 2020-11-05 at 14 19 11" src="https://user-images.githubusercontent.com/14192054/98242223-cb281080-1f74-11eb-950f-32a9e984e8a5.png">

Created issue https://github.com/Automattic/wp-calypso/issues/47147